data/sql/updates/pending_db_world/rev_1629322815778584087.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629322815778584087');
+
+UPDATE gameobject SET position_x = 4217.04, position_y = 906.04, position_z = -8.08 WHERE GUID = 48120;
+UPDATE gameobject SET position_x = 4222.18, position_y = 914.03, position_z = -7.44 WHERE GUID = 63507;
```

Comment:
Ty for PR, please add backticks =)

If you want to learn more about code standards you can read the SOP on the wiki https://www.azerothcore.org/wiki/standard-operating-procedure :D